windows/ReactTestApp/MainPage.cpp

```diff
@@ -76,4 +78,43 @@ namespace winrt::ReactTestApp::implementation
         ReactRootView().ComponentName(s.as<ComponentViewModel>()->AppKey());
     }
 
+    void MainPage::OpenReactMenu(IInspectable const &, RoutedEventArgs)
+    {
+        ReactButton().Flyout().ShowAt(ReactButton());
+    }
+
+    MenuFlyoutItem MainPage::GetComponentMenuButton(::ReactTestApp::Component &component)
+    {
+        hstring componentDisplayName = to_hstring(component.displayName.value_or(component.appKey));
+        hstring appKey = to_hstring(component.appKey);
+        ReactTestApp::ComponentViewModel newComponent =
+            winrt::make<ComponentViewModel>(appKey, componentDisplayName);
+
+        MenuFlyoutItem newMenuItem;
+        newMenuItem.Text(newComponent.DisplayName());
+        newMenuItem.CommandParameter(newComponent);
+        newMenuItem.Click({this, &MainPage::SetReactComponentName});
+        return newMenuItem;
+    }
+
+    void MainPage::SetUpTitleBar()
+    {
+        // Set close, minimize and maximize icons background to transparent
+        auto appView = ApplicationView::GetForCurrentView().TitleBar();
+        appView.ButtonBackgroundColor(winrt::Windows::UI::Colors::Transparent());
+        appView.BackgroundColor(winrt::Windows::UI::Colors::Transparent());
```

Comment:
Nit: You can add `using winrt::Windows::UI::Colors;` up top:

```suggestion
        appView.ButtonBackgroundColor(Colors::Transparent());
        appView.BackgroundColor(Colors::Transparent());
```